server/reminders/reminderApi.ts

```diff
@@ -0,0 +1,137 @@
+import crypto from 'crypto';
```

Comment:
this file is moved from server/reminderApi.ts
I've added `publicCreateReminderHandler` and logic around verification